docs/sql-reference/13-sql-reference/41-access-control-privileges.md

```diff
@@ -99,10 +103,28 @@ Please note that you can use the [USE DATABASE](../14-sql-commands/00-ddl/10-dat
 
 ## Stage Privileges
 
-| Privilege | Description                                              |
-|:----------|:---------------------------------------------------------|
-| SUPER     | Lists stages. Creates, drops or removes a stage.         |
-| ALL       | Grants all the privileges for the specified object type. |
+| Privilege | Description                                                                                                       |
+|:----------|:------------------------------------------------------------------------------------------------------------------|
+| WRITE     | Write into a stage. For example, copy into a stage, presign upload or removes a stage                             |
```

Comment:
Writes data to a stage, such as unloading with COPY INTO, uploading via a presigned URL to a stage, or deleting files from a stage.